src/Library/demos/Drop Down/main.blp

```diff
@@ -0,0 +1,71 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Drop Down");
+  description:_("choose an item from a list of options");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+    valign: center;
+
+    Box  {
+      orientation: horizontal;
+      halign: center;
+      valign: center;
+      spacing: 12;
+
+      Label {
+        halign: start;
+        label: _("Selection:");
+      }
+
+      DropDown drop_down {
+        halign: end;
+        width-request: 100;
+        model: StringList {
+          strings ["Item 1", "Item 2", "Item 3", "Item 4"]
```

Comment:
Maybe fruits and vegetables?

Reply:
Also, since you removed the simple searchable example, can you make this one searchable?

I think it's good for users to see how to make a simple Dropdown with StringList searchable because it's really not obvious